dace/transformation/passes/transient_reuse.py

```diff
@@ -151,17 +151,6 @@ def apply_pass(self, sdfg: SDFG, _) -> Optional[Set[str]]:
                                 if edge.data.data == old:
                                     edge.data.data = new
 
-        # clean up the arrays
-        for a in list(sdfg.arrays):
```

Comment:
I guess we have an API call for this piece of code, but why was it removed completely?